submissions/divineOmozokpia/HTML-Forms/contactpage.html

```diff
@@ -5,6 +5,13 @@
         <title>Contact Form</title>
     </head>
     <body>
+        <nav>
+            <ul>
+                <li><a href="../HTML/product.html">Product</a></li>
+                <li><a href="../html-forms/contactpage.html">Contact Us</a></li>
```

Comment:
Your folder name is HTML-Forms. whereas you are using html-forms in the link. Are you sure it is redirecting correctly?